lib/ts/recipe/session/components/themes/accessDeniedScreenTheme/index.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/jsx-no-literals */
+import React, { Fragment } from "react";
+import { withOverride } from "../../../../../components/componentOverride/withOverride";
+import { hasFontDefined } from "../../../../../styles/styles";
+import { useTranslation } from "../../../../../translation/translationContext";
+import { AccessDeniedThemeProps } from "../../../types";
+import { ThemeBase } from "../themeBase";
+
+export const AccessDeniedTheme = withOverride<React.FC<AccessDeniedThemeProps>>(
+    "SessionAccessDenied",
+    function AccessDeniedThemeScreen(props) {
+        const t = useTranslation();
+        const hasFont = hasFontDefined(props.config.rootStyle);
+
+        return (
+            <ThemeBase
+                loadDefaultFont={!hasFont}
+                userStyles={[props.config.rootStyle, props.config.accessDeniedScreen.style]}>
+                <Fragment>
+                    <div data-supertokens="accessDeniedTitle">{t("ACCESS_DENIED")}</div>
```

Comment:
We already have title styles, please use those (check the `INVALID` state of the `veryEmailLinkClicked` theme component.